lib/Model.php

```diff
@@ -1708,30 +1708,30 @@ public static function last(/* ... */)
      *
      * @throws RecordNotFound if no options are passed or finding by pk and no records matched
      *
-     * @return static|static[]|null
+     * @return Model|Model[]|null
```

Comment:
Why change this? `static` was correct--what gets returned is the late-static-bound subclass of model. So `Model` isn't wrong, exactly, but it sort of buries the lede, doesn't it?

Reply:
Ok reverted